core/src/main/scala/org/apache/spark/sql/crossdata/catalog/persistent/PostgreSQLXDCatalog.scala

```diff
@@ -210,7 +216,7 @@ class PostgreSQLXDCatalog(sqlContext: SQLContext, override val catalystConf: Cat
     if (!resultSet.isBeforeFirst) {
       resultSet.close()
       val prepped = connection.prepareStatement(
-        s"""|INSERT INTO $db.$table (
+        s"""|INSERT INTO $db.$tableWithTableMetadata (
```

Comment:
`$db.$tableWithTableMetadata` is repeated several times, isn't it worth of be encapsulated in a method or function?